### ISSUE TITLE
tsconfig.json `include`s from node_modules not rewritten

### ISSUE THREAD

**Issue by robaca** [stryker-mutator/stryker-js#3281]
Currently we are referencing d.ts types from a shared library in our local projects like here:

```json
{
    "extends": "../../../tsconfig-base.json",
    "compilerOptions": {
        "target": "es6",
        "lib": [ "es6", "dom" ],
    },
    "include": [
        "./**/*",
        "./**/*.json",
        "../../../node_modules/self-service-server/lib/main/shared/@types/**/*.d.ts",
    ]
}
```
When copying these files into the sandbox, the paths are not altered, so that they point to non-existent subdirectories and `tsc -b` is failing.

As a workaround, we added them like this:
```json
{
    "include": [
        "../../../node_modules/self-service-server/lib/main/shared/@types/**/*.d.ts",
        "../../../../../node_modules/self-service-server/lib/main/shared/@types/**/*.d.ts",
    ]
}
```

Please alter paths and add additional `../../`to the entries. This might also make sense for other directives (typeRoots, excludes, ...).

**Stryker config**

```json
{
    "$schema": "./node_modules/@stryker-mutator/core/schema/stryker-schema.json",
    "_comment": "Stryker Mutator config",
    "packageManager": "yarn",
    "reporters": [
        "html",
        "clear-text",
        "progress"
    ],
    "mutate": [
        "src/main/**/*.ts",
        "!src/main/**/*Spec.ts"
    ],
    "ignorePatterns": ["lib", "build", "www"],
    "testRunner": "mocha",
    "testRunnerNodeArgs": [
        "--require",
        "esm"
    ],
    "mochaOptions": {
        "extension": [
            "ts",
            "tsx"
        ],
        "spec": [
            "src/**/*Spec.ts"
        ],
        "require": [
            "src/test/ts-hook",
            "ignore-styles",
            "mocha-steps"
        ],
        "package": "./package.json",
        "ui": "bdd",
        "async-only": false,
        "grep": ".*"
    },
    "coverageAnalysis": "perTest",
    "checkers": [
        "typescript"
    ],
    "tsconfigFile": "src/test/tsconfig.json"
}
```

**Test runner config**

.mocharc.js
```js
'use strict';
module.exports = {
    reporter: 'mocha-better-spec-reporter',
    file: ['src/test/setupEnvironment.ts'],
    extension: ['.ts'],
    require: [
        'esm',
        'src/test/ts-hook',
        'ignore-styles',
        'mocha-steps'
    ],
    recursive: true,
    timeout: '10s',
    exit: true,
};
```

**Stryker environment**

```
├─ @stryker-mutator/api@5.5.0
├─ @stryker-mutator/core@5.5.0
│  ├─ @stryker-mutator/api@5.5.0
│  ├─ @stryker-mutator/instrumenter@5.5.0
│  ├─ @stryker-mutator/util@5.5.0
├─ @stryker-mutator/instrumenter@5.5.0
│  ├─ @stryker-mutator/api@5.5.0
│  ├─ @stryker-mutator/util@5.5.0
├─ @stryker-mutator/mocha-runner@5.5.0
│  ├─ @stryker-mutator/api@5.5.0
│  ├─ @stryker-mutator/util@5.5.0
├─ @stryker-mutator/typescript-checker@5.5.0
│  ├─ @stryker-mutator/api@5.5.0
│  ├─ @stryker-mutator/util@5.5.0
├─ @stryker-mutator/util@5.5.0
```

```
├─ mocha@8.4.0
├─ esm@3.2.25
├─ ts-node@9.1.1
```

**Test runner environment**

```shell
yarn run mocha --exit 'src/**/*Spec.ts'
```

**Your Environment**

| software         | version(s)
| ---------------- | -------
| node             | 16.13.0
| yarn              | 1.22.10
| Operating System |
|NixOS          | unstable

**Comment by nicojs** [comment]
Thanks for opening this issue. As a workaround, could to try to run with `--inPlace`?

**Comment by nicojs** [comment]
Fix released in v5.5.1
